src/Android/Autofill/AutofillHelpers.cs

```diff
@@ -152,23 +153,24 @@ public static class AutofillHelpers
             "androidapp://com.oneplus.applocker",
         };
 
-        public static async Task<List<FilledItem>> GetFillItemsAsync(Parser parser, ICipherService cipherService)
+        public static async Task<List<FilledItem>> GetFillItemsAsync(Parser parser, ICipherService cipherService, IUserVerificationService userVerificationService)
         {
+            var allowReprompt = !await userVerificationService.HasMasterPasswordAsync();
```

Comment:
I think `hasMasterPassword` would be a more appropriate name for this so it makes more sense during use below, thoughts?